crates/planus/src/lib.rs

```diff
@@ -37,9 +37,9 @@ pub use crate::{
 #[doc(hidden)]
 pub const fn check_version_compatibility(s: &str) {
     match s.as_bytes() {
-        b"planus-0.4.0" => (),
+        b"planus-1.0.0" => (),
         _ => panic!(
-            "Your generated code is out of date, please regenerate using planus version 0.4.0"
+            "Your generated code is out of date, please regenerate using planus version 1.0.0"
```

Comment:
Should this do `starts_with`, or warn instead of panic for minor/patch versions? I'm not sure what's most convenient.

Reply:
It's a compile time panic and it has been there since forever.

Reply:
It's there because we want to guarantee forwards/backwards compatibility of the API part of planus and of the API of the generated code, but we consider the internal API between planus and the generated code an implementation detail.